tests/MongoDB.Driver.Tests/AggregateFluentTests.cs

```diff
@@ -196,6 +196,71 @@ public void ChangeStream_should_add_the_expected_stage_when_options_is_null()
             result.Should().Be(1);
         }
 
+        [SkippableFact]
+        public void Function_should_return_expected_result()
+        {
+            RequireServer.Check().Supports(Feature.AggregateFunction);
+
+            var client = CreateClient();
+            var databaseName = "test";
+            var collectionName = "collection";
+            DropCollection(client, databaseName, collectionName);
+            var collection = client.GetDatabase(databaseName).GetCollection<BsonDocument>(collectionName);
+            var players = new[]
+            {
+                BsonDocument.Parse("{ _id : 1, name : 'Miss Cheevous',  scores : [10, 5, 10] }"),
+                BsonDocument.Parse("{ _id : 2, name : 'Miss Ann Thrope', scores : [10, 10, 10] }"),
+                BsonDocument.Parse("{ _id : 3, name : 'Mrs. Eppie Delta', scores : [9, 8, 8] }")
+            };
+            collection.InsertMany(players);
+
+            var isFoundBodyFunction = "function(name) { return hex_md5(name) == '15b0a220baa16331e8d80e15367677ad' }";
+            var messageBodyFunction = "function(name, scores) { let total = Array.sum(scores); return `Hello ${ name}. Your total score is ${ total}.` }";
```

Comment:
Rename to: `messageFunction` (rename `isFoundFunction` below first)

Reply:
done